Core/src/org/sleuthkit/autopsy/communications/CVTEvents.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2017-18 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.communications;
+
+import com.google.common.eventbus.EventBus;
+
+/**
+ * Provide he singleton EventBus.
+ */
+final class CVTEvents {
+
+    private final static EventBus cvtEventBus = new EventBus();
+
+    public static EventBus getCVTEventBus() {
```

Comment:
See previous comment about public methods on package access classes. Also, I'm curious, why have the event carry a reference to the event bus?

Reply:
Yes, I will do a sweep for public methods.  This is not an event, i just provides static access to the eventbus.  Maybe it could go somewhere else...